test/hotspot/jtreg/compiler/loopopts/superword/TestCyclicDependency.java

```diff
@@ -28,7 +28,12 @@
  * @summary Writing forward on array creates cyclic dependency
```

Comment:
You could probably add the bug number of this bug to the `@bug` in the line above this one.